test/Confluent.Kafka.IntegrationTests/Tests/AddBroker.cs

```diff
@@ -17,96 +17,55 @@
 #pragma warning disable xUnit1026
 
 using System;
-using System.Text;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Xunit;
 
 
-/*
 namespace Confluent.Kafka.IntegrationTests
 {
-    public static partial class Tests
+    public partial class Tests
     {
         /// <summary>
-        ///     Test that produces a message then consumes it.
+        ///     Test the AddBroker method works.
         /// </summary>
+        /// <remarks>
+        ///     Assumes broker v0.10.0 or higher:
+        ///     https://github.com/edenhill/librdkafka/wiki/Broker-version-compatibility
+        ///     A metadata request is used to check if there is a broker connection.
+        /// </remarks>
         [Theory, MemberData(nameof(KafkaParameters))]
-        public static void AddBrokers(string bootstrapServers, string singlePartitionTopic, string partitionedTopic)
+        public void AddBrokers(string bootstrapServers)
         {
-            // This test assumes broker v0.10.0 or higher:
-            // https://github.com/edenhill/librdkafka/wiki/Broker-version-compatibility
+            var producerConfig = new ProducerConfig { BootstrapServers = "unknown" };
```

Comment:
Albeit unlikely, unknown might resolve to something for other developer's environments, or simply stall while trying to resolve.
Suggest you use "localhost:65533" or something that is unlikely to be bound, if you just want a broker that will not respond.

Reply:
good point